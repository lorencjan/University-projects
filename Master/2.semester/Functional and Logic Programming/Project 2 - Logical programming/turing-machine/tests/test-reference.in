S a B a
B a B b
B b B R
B c B a
B c F c
B c B a
aaacaa
