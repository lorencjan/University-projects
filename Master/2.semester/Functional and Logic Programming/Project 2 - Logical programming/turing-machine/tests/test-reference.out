Saaacaa
Baaacaa
Bbaacaa
bBaacaa
bBbacaa
bbBacaa
bbBbcaa
bbbBcaa
bbbFcaa
